boards/samr21-xpro/include/gpio_params.h

```diff
@@ -37,12 +37,14 @@ static const  saul_gpio_params_t saul_gpio_params[] =
     {
         .name = "LED(orange)",
         .pin = LED0_PIN,
-        .mode = GPIO_OUT
+        .mode = GPIO_OUT,
+        .flags = SAUL_GPIO_INVERTED,
```

Comment:
IIUC the logic, you should add SAUL_GPIO_CLEAR (or SAUL_GPIO_INIT_CLEAR) to ensure the pin initialized cleared ? Otherwise it's set after initialization (default GPIO behavior on this board).

Reply:
I think changing the initial value belongs to a different commit. So I prefer not include it in this PR.